flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -231,6 +231,11 @@ public Builder writeParallelism(int newWriteParallelism) {
       return this;
     }
 
+    public Builder properties(Map<String, String> properties) {
```

Comment:
Nit: Can we add a javadoc here similar to the other ones? This might help me understand the flow better for non-SQL users. Users shouldn't pass in the run-time properties map, correct?

We might want to be somewhat more specific about which `properties` should come here.

Reply:
Adding a javadoc is a good note.
As described above, we have another PR #3998 processing non-SQL case, that is, Java api.